server/src/internalClusterTest/java/org/opensearch/gateway/remote/RemotePublicationConfigurationIT.java

```diff
@@ -97,23 +98,26 @@ public Settings.Builder remotePublishConfiguredNodeSetting() {
             .put(stateRepoSettingsAttributeKeyPrefix + prefixModeVerificationSuffix, true)
             .put(REMOTE_CLUSTER_STATE_ENABLED_SETTING.getKey(), true)
             .put("node.attr." + REMOTE_STORE_ROUTING_TABLE_REPOSITORY_NAME_ATTRIBUTE_KEY, ROUTING_TABLE_REPO_NAME)
-            .put(routingTableRepoTypeAttributeKey, ReloadableFsRepository.TYPE)
+            .put(routingTableRepoTypeAttributeKey, FsRepository.TYPE)
             .put(routingTableRepoSettingsAttributeKeyPrefix + "location", segmentRepoPath);
         return builder;
     }
 
     public Settings.Builder remoteWithRoutingTableNodeSetting() {
         // Remote Cluster with Routing table
+
         return Settings.builder()
             .put(
-                buildRemoteStoreNodeAttributes(
+                remoteStoreClusterSettings(
                     REPOSITORY_NAME,
                     segmentRepoPath,
+                    ReloadableFsRepository.TYPE,
                     REPOSITORY_2_NAME,
                     translogRepoPath,
+                    ReloadableFsRepository.TYPE,
                     REPOSITORY_NAME,
                     segmentRepoPath,
-                    false
+                    ReloadableFsRepository.TYPE
```

Comment:
do you know why segment repo is repeated here?

Reply:
this is the parameter for routingTable. the same path as that of SEGMENT is used.